Xamarin.Forms.Core/Element.cs

```diff
@@ -26,9 +25,9 @@ public static void SetMenu(BindableObject bindable, Menu menu)
 
 		internal static readonly ReadOnlyCollection<Element> EmptyChildren = new ReadOnlyCollection<Element>(new Element[0]);
 
-		public static readonly BindableProperty ClassIdProperty = BindableProperty.Create("ClassId", typeof(string), typeof(View), null);
+		public static readonly BindableProperty AutomationIdProperty = BindableProperty.Create(nameof(AutomationId), typeof(string), typeof(View), null);
 
-		string _automationId;
+		public static readonly BindableProperty ClassIdProperty = BindableProperty.Create(nameof(ClassId), typeof(string), typeof(View), null);
```

Comment:
Why is ClassId a `View` BP instead of an `Element` BP? I made AutomationId match it, but I think they should belong to `Element`, no?

Reply:
Yes, I think it should be `Element`.

Reply:
Done.